mobile/components/Menu.js

```diff
@@ -30,76 +30,143 @@ const styles = {
+                            this.context.drawer.close();
+                        }}
+            style={styles.menuItem}
+          >
+            <View style={styles.iconContainer}>
+              <Icon name="home" size={20} color="#337ab7" />
+            </View>
+            <Text style={styles.menuItemText}>Home</Text>
+          </TouchableOpacity>
 
-    <Collapsible title="Modules" headerIcon="graduation-cap" style={styles.menuItem}>
-      <TouchableOpacity onPress={Actions.home} style={styles.menuItem}>
-        <View style={[styles.iconContainer, { marginLeft: 30 }]}>
-          <Icon name="check" size={20} color="#337ab7" />
-        </View>
-        <Text style={styles.menuItemText}>Assign</Text>
-      </TouchableOpacity>
-    </Collapsible>
+          <Collapsible title="CHW" headerIcon="users" style={styles.menuItem}>
+            <View>
+              <TouchableOpacity
+                onPress={() => {
+                                    Actions.home();
+                                    this.context.drawer.close();
+                                }}
+                style={styles.menuItem}
+              >
+                <View style={[styles.iconContainer, { marginLeft: 30 }]}>
+                  <Icon name="plus" size={20} color="#337ab7" />
+                </View>
+                <Text style={styles.menuItemText}>Add CHW</Text>
+              </TouchableOpacity>
+              <TouchableOpacity
+                onPress={() => {
+                                    Actions.home();
+                                    this.context.drawer.close();
+                                }}
+                style={styles.menuItem}
+              >
+                <View style={[styles.iconContainer, { marginLeft: 30 }]}>
+                  <Icon name="list" size={20} color="#337ab7" />
+                </View>
+                <Text style={styles.menuItemText}>CHW List</Text>
+              </TouchableOpacity>
+            </View>
+          </Collapsible>
 
-    <Collapsible title="Incharge" headerIcon="user" style={styles.menuItem}>
-      <View>
-        <TouchableOpacity onPress={Actions.home} style={styles.menuItem}>
-          <View style={[styles.iconContainer, { marginLeft: 30 }]}>
-            <Icon name="plus" size={20} color="#337ab7" />
-          </View>
-          <Text style={styles.menuItemText}>Add Incharge</Text>
-        </TouchableOpacity>
-        <TouchableOpacity onPress={Actions.incharges} style={styles.menuItem}>
-          <View style={[styles.iconContainer, { marginLeft: 30 }]}>
-            <Icon name="list" size={20} color="#337ab7" />
-          </View>
-          <Text style={styles.menuItemText}>Incharge List</Text>
-        </TouchableOpacity>
-      </View>
-    </Collapsible>
+          <Collapsible title="Modules" headerIcon="graduation-cap" style={styles.menuItem}>
+            <TouchableOpacity
+              onPress={() => {
+                                Actions.home();
```

Comment:
Can we make this a single function which will take the name of a screen?

The reason is, we should not be able to forget about closing drawer.